docs/UserGuide.md

```diff
@@ -156,9 +156,20 @@ Format: `event -n EVENT_NAME -d EVENT_DATE (yyyy-mm-dd) -l LOCATION -a ATTENDEES
 <box type="tip" seamless>
 
 **Tip:** All parameters `EVENT_NAME`, `EVENT_DATE`, `LOCATION` must be present but `ATTENDEES` is optional
+
 **Tip:** Indexes supplied to the `ATTENDEES` parameter must be based on existing contacts indexing in the Address Book
 </box>
 
+### Updating an event: `update`
+
+Updates the details of an existing event in the address book. 
+
+Format: `update -i INDEX -n NEW_NAME -d NEW_DATE -l NEW_LOCATION -a NEW_ATTENDEES_INDICES -r REMOVED_ATTENDEES_INDICES`
```

Comment:
Perhaps we can add [ ] for the fields that are optional, to standardise with the `edit` command